packages/render-fidelity-tools/src/components/renderers/dspbr-pt-viewer.ts

```diff
@@ -82,11 +82,11 @@ export class PathtracingViewer extends LitElement {
     this[$camera] = new PerspectiveCamera(45, this[$canvas]!.width/this[$canvas]!.height, 0.01, 1000);
     this[$renderer] = new PathtracingRenderer({ canvas: this[$canvas]!});
 
-    const renderer = this[$renderer];
-    renderer.pixelRatio = 1.0; 
+    this[$renderer].pixelRatio = 1.0; 
     // this.renderer.iblRotation = 180.0;
-    renderer.exposure = 1.0;
-    renderer.maxBounces = 8;
+    this[$renderer].exposure = 1.0;
+    this[$renderer].maxBounces = 8;
+    this[$renderer].tonemapping = "AcesFilm";
```

Comment:
Is this the three.js ACES tone mapping function, or a different implementation? I notice the difference, which I think will bring it more in line with the other renderers, so that's great.

Reply:
Yes, it's the three (and sample viewer) ACES function